manifests/c/CodeSector/TeraCopy/3.9.0/CodeSector.TeraCopy.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CodeSector.TeraCopy
+PackageVersion: 3.9.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://codesector.s3.us-west-000.backblazeb2.com/teracopy3.9.exe
+  InstallerSha256: 9DFA57C6DD935F72E7E8254146CD143528FE962D2C2F518C2D6511D4D1150A3A
+  InstallerSwitches:
+    Silent: /q
+    SilentWithProgress: /qb
+  UpgradeBehavior: install
+  ProductCode: '{952ED35A-74C3-4204-8F01-986D8711B41D}'
```

```suggestion
  ProductCode: '{F8B0BB18-B1E6-4821-8C5B-883AA5DE3EEA}'
```

> i do not understand whey the product code is coming from, and why it is different from the earlier version here. where did you get this product code `F8B0BB18-B1E6-4821-8C5B-883AA5DE3EEA`?

It's taken from the Windows Registry.

---

okay. i wasnt ware of this. thanks